### PR TITLE
Feature/ys/#123

### LINKED ISSUE #123: Ayarlar sayfası
**Description**

EXPANSION OLACAK

- sol üstte ayarlar sliver olacak şekilde hep yapılacak
- en üstte takımı gösteren bir alan ve buraya batığında takımın detalyaırnı görecek
- change language basınca bir bottom sheet ile aktif uygulama dili gösterilecek ve diğer dilla seçim yapıp kaydet derse o dile geçilecek
- bildiimler kapılı ise changen otificaton gözükecek ve yine basılıdğında app dışında ayalra gidilip açılmasına bkaılacak
- app settings yan sayfaya geçiecek uygulama versiyon, marketing sitesi ve detaylar gözükecek proje hakkında
- get supprot basınca doğrudan bir sheet açılacak ve iki özellik verilecek bir mail atma özelliği burada samete ve bana yollandırlaacak sekilde hazılanır iki tweet gödnerme özelliği burada hazır bir template yapılıp o bağlanacak
- favorilerim tabı olacak ve mekanlar göüzekecek burada 

**Screen Preview**
image

### PR DESCRIPTION
**Pull request by mysCod3r**
closed #123,
| | |
|-|-|
|image|image

### CODE REVIEW

File: lib/features/settings_module/settings/subview/notification_permission_checkbox.dart [hunk 1]
```diff
@@ -5,6 +5,7 @@ import 'package:vbaseproject/product/init/language/locale_keys.g.dart';
 import 'package:vbaseproject/product/model/enum/approve_dialog_type.dart';
 import 'package:vbaseproject/product/package/settings/custom_app_settings.dart';
 import 'package:vbaseproject/product/widget/dialog/approve_dialog.dart';
+import 'package:vbaseproject/product/widget/general/title/general_body_title.dart';
 
 class NotificationPermissionView extends StatefulWidget {
```
Comment (burakJs) [inline lib/features/settings_module/settings/subview/notification_permission_checkbox.dart]: `final` and `@immutable`

File: lib/features/v2/settings/view/widget/app_about_widget.dart [hunk 2]
```diff
@@ -0,0 +1,24 @@
+part of '../settings_view.dart';
+
+final class _AppAboutWidget extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/app_about_widget.dart:4]: `@immutable`

File: lib/features/v2/settings/view/widget/change_language_widget.dart [hunk 3]
```diff
@@ -0,0 +1,22 @@
+part of '../settings_view.dart';
+
+final class _ChangeLanguageWidget extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/change_language_widget.dart:4]: `@immutable`

File: lib/features/v2/settings/view/widget/change_notification_widget.dart [hunk 4]
```diff
@@ -0,0 +1,15 @@
+part of '../settings_view.dart';
+
+final class _ChangeNotificationWidget extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/change_notification_widget.dart:4]: `@immutable`

File: lib/features/v2/settings/view/widget/contact_us_widget.dart [hunk 5]
```diff
@@ -0,0 +1,94 @@
+part of '../settings_view.dart';
+
+final class _ContactUsWidget extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/contact_us_widget.dart:4]: `@immutable`

File: lib/features/v2/settings/view/widget/contact_us_widget.dart [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    return const GeneralExpansionTile(
+      pageTitle: LocaleKeys.settings_contactTitle,
+      children: [
+        _ContactUsGridView(),
+      ],
+    );
+  }
+}
+
+final class _ContactUsGridView extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/contact_us_widget.dart:19]: `@immutable`

File: lib/features/v2/settings/view/widget/contact_us_widget.dart [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+        crossAxisCount: 2,
+        mainAxisExtent: WidgetSizes.spacingXxlL13,
+      ),
+      itemBuilder: (context, i) =>
+          _ContactUsCard(model: ContactModel.dummyModels[i]),
+    );
+  }
+}
+
+final class _ContactUsCard extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/contact_us_widget.dart:39]: `@immutable`

File: lib/features/v2/settings/view/widget/contact_us_widget.dart [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+            icon: Icons.mail_outline,
+            onTap: () => model.mail.ext.launchEmail,
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+final class _UserImage extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/contact_us_widget.dart:69]: `@immutable`

File: lib/features/v2/settings/view/widget/contact_us_widget.dart [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return CircleAvatar(
+      radius: CustomCircleRadius.large + CustomCircleRadius.small,
+      backgroundImage: NetworkImage(imageUrl),
+    );
+  }
+}
+
+final class _ContactTile extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/contact_us_widget.dart:84]: `@immutable`

File: lib/features/v2/settings/view/widget/developers_widget.dart [hunk 10]
```diff
@@ -0,0 +1,31 @@
+part of '../settings_view.dart';
+
+final class _DevelopersWidget extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/developers_widget.dart:4]: `@immutable`

File: lib/product/widget/dropdown/language_dropdown_widget.dart [hunk 11]
```diff
@@ -0,0 +1,47 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:kartal/kartal.dart';
+import 'package:vbaseproject/core/init/core_localize.dart';
+import 'package:vbaseproject/product/utility/decorations/custom_radius.dart';
+import 'package:vbaseproject/product/widget/general/title/index.dart';
+
+final class LanguageDropdownWidget extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/dropdown/language_dropdown_widget.dart:9]: `@immutable`

File: lib/product/widget/general/general_expansion_tile.dart [hunk 12]
```diff
@@ -0,0 +1,35 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:kartal/kartal.dart';
+import 'package:vbaseproject/product/utility/padding/page_padding.dart';
+import 'package:vbaseproject/product/widget/general/index.dart';
+
+class GeneralExpansionTile extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/general/general_expansion_tile.dart]: `final` and `@immutable`

File: lib/features/v2/settings/view/widget/developers_widget.dart [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+        title: GeneralContentTitle(
+          value: LocaleKeys.developers_title.tr(),
+          fontWeight: FontWeight.bold,
+        ),
+        subtitle: GeneralContentSubTitle(
+          value: LocaleKeys.settings_seeDevelopers.tr(),
+        ),
+        contentPadding: const PagePadding.generalAllNormal(),
+        leading: const Icon(
+          Icons.groups_2_outlined,
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/developers_widget.dart]: from AppIcons

File: lib/features/v2/settings/view/widget/app_about_widget.dart [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+      elevation: 0,
+      shape: context.border.roundedRectangleAllBorderNormal
+          .copyWith(side: const BorderSide()),
+      child: ListTile(
+        contentPadding: const PagePadding.generalAllNormal(),
+        title: GeneralBodyTitle(
+          LocaleKeys.settings_aboutTitle.tr(),
+          fontWeight: FontWeight.bold,
+        ),
+        trailing: const Icon(Icons.chevron_right),
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/app_about_widget.dart]: from AppIcons

File: lib/product/widget/dropdown/language_dropdown_widget.dart [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+    );
+  }
+
+  InputDecoration get _decoration => InputDecoration(
+        border: _border,
+        focusedBorder: _border,
+      );
+
+  OutlineInputBorder get _border => const OutlineInputBorder(
+        borderSide: BorderSide(strokeAlign: 0.1),
```
Comment (BerkSartik) [inline lib/product/widget/dropdown/language_dropdown_widget.dart]: BorderSide class çıkarılabilir

File: lib/features/v2/settings/view/widget/contact_us_widget.dart [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+final class _UserImage extends StatelessWidget {
+  const _UserImage({required this.imageUrl});
+  final String imageUrl;
+  @override
+  Widget build(BuildContext context) {
+    return CircleAvatar(
+      radius: CustomCircleRadius.large + CustomCircleRadius.small,
+      backgroundImage: NetworkImage(imageUrl),
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/contact_us_widget.dart]: from CustomNetworkImage

File: lib/features/v2/settings/view/widget/developers_widget.dart [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+          size: WidgetSizes.spacingXxl6,
+        ),
+        trailing: const Icon(Icons.chevron_right),
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/developers_widget.dart]: from AppIcons

File: lib/features/v2/settings/view/widget/app_about_widget.dart [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+        onTap: () {},
```
Comment (burakJs) [inline lib/features/v2/settings/view/widget/app_about_widget.dart]: get from member of class

**Pull request merged**